vsdkx/addon/tracking/processor.py

```diff
@@ -100,18 +100,16 @@ def _box_counter(self, boxes: np.ndarray) -> tuple:
                     if not self._bidirectional_mode:
                         event_counter += 1
 
-                if self._bidirectional_mode and \
-                        not self._trackable_obj.position:
-                    if self._trackable_obj.tracked_number >= self._min_appearance:
```

Comment:
If we remove this statement, then I think we no longer use `min_appearance` parameter and we should deprecate it properly, remove it from constructor and from configurations ? 

what do you think

Reply:
Nice catch! On it 🚀